contracts/build.rs

```diff
@@ -28,9 +28,8 @@ fn main() {
             .add_deployment(
                 4,
                 addr("0xBA12222222228d8Ba445958a75a0704d566BF2C8"),
-                Some(tx(
-                    "0x5fe65a242760f7f32b582dc402a081791d57ea561474617fcd0e763c995cfec7",
-                )),
+                // <https://rinkeby.etherscan.io/tx/0x5fe65a242760f7f32b582dc402a081791d57ea561474617fcd0e763c995cfec7>
+                Some(DeploymentInformation::BlockNumber(8441702)),
```

Comment:
Is it more convenient to put the block number with a comment linking to the tx hash or just set this thing up with the tx hash as it's deployment info and fetch the rest. I see we are saving an eth call so that's nice.

Reply:
> I see we are saving an eth call so that's nice.

That's my main motivation. It also saves us one failure branch.